MemorialHouse/MHPresentation/MHPresentation/Source/Design/MHRegisterView.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+
+final class MHRegisterView: UIView {
+    // MARK: UI Components
+    let registerTextField: UITextField = {
+        let registerFont = UIFont.ownglyphBerry(size: 12)
+        
+        let textField = UITextField()
+        textField.font = registerFont
+        
+        var attributedText = AttributedString(stringLiteral: "ex) 영현")
```

Comment:
확실히 예뻐졌네요 ^ㅇ^

Reply:
인정합니다. 빛

Reply:
```suggestion
        var attributedText = AttributedString(stringLiteral: "ex) 효준")
```

음 확실히 "효준"이 더 예뻐보이네요 ^^ 바꾸시죠

Reply:
Resolve 처리하겠습니다 수고요 ㅎㅎ

<img width="942" alt="image" src="https://github.com/user-attachments/assets/a3c9335f-8cfa-441c-bd47-ae7e715f5895">

Reply:
제..의견..무시.........흑흑